polyphemus/lib/commands.rb

```diff
@@ -268,7 +269,7 @@ def execute
           # Magma does not support delete right now, so we restrict the models, but we can unlink and
           # delete the files in metis
           restrict!(patient, delete_metis_files: true)
-        elsif should_be_restricted && ! patient['restricted']
+        elsif should_be_restricted
           restrict!(patient)
         elsif ! should_be_restricted && patient['restricted']
```

Comment:
Sorry, old PR -- but should this check also not check `patient['restricted']`, and just always rely on `should_be_restricted`?